dispatches/models/renewables_case/wind_battery_LMP.py

```diff
@@ -72,19 +72,19 @@ def wind_battery_om_costs(m):
 def initialize_mp(m, verbose=False):
     outlvl = idaeslog.INFO if verbose else idaeslog.WARNING
 
-    m.fs.windpower.initialize(outlvl=outlvl)
+    # m.fs.windpower.initialize(outlvl=outlvl)
```

Comment:
Why are the calls to initialize being removed?  Is this intentional?  If so, should they just be deleted?

Reply:
@xiangao1 were the initialize calls removed for speed-up? this `initialize_mp` function is only called once during `build_multi_period_model`, at the the first timestep, after that, the model is cloned. So I don't think these calls are going to impact speed at all, so I'm inclined to leave them for completion sake.